app/controllers/providers/allocations_controller.rb

```diff
@@ -9,6 +10,13 @@ def index; end
 
   private
 
+    def build_allocations
+      @allocations = []
+      @allocations <<  { provider_name: "Enfield County School for Girls", status: "Confirm your choice", status_colour: "grey", action: "" }
+      @allocations <<  { provider_name: "London Academy", status: "NOT REQUESTED", status_colour: "red", action: "Change" }
+      @allocations <<  { provider_name: "University of East Anglia", status: "REQUESTED", status_colour: "green", action: "Change" }
```

Comment:
`status_colour` is just a placeholder for the time being. This will be abstracted out into a helper method when we integrate the API and the view. The idea is that we would pass the status to the helper method which in turn would return a CSS class name to style the status appropriately as we do for course status.